### PR TITLE
increased timeout delay to 15s

### PR DESCRIPTION
**Pull request by ajenner**
Timeout increased

### CODE REVIEW

File: panacea/web/static/js/app.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ async function submitFile() {
       credentials: 'same-origin',
       headers: new Headers({authorization: apiAccessToken})
     });
-
+
```
Comment (22a) [inline panacea/web/static/js/app.js]: wat's this then

**Comment by 22a** [comment]
what's happening to `erlang-solutions_1.0_all.deb` here?

File: panacea/web/static/js/app.js (hunk 1 above)
Comment (ajenner) [reply panacea/web/static/js/app.js]: Vital whitespace adjustments

**Review by c-brenn — APPROVED** [review]
looks good to me - nice one.

**Comment by c-brenn** [comment]
squash and merge at your leisure @ajenner

**Pull request merged**